.devcontainer/bootstrap.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Check if the SmartOrderApi directory exists and is a Git repository
+for project in SmartOrderApi daemon-dotnet; do
+    [[ -d "../$project/.git" ]] && continue;
+    git clone "$GITHUB_DAEMONTECHTOOLS/$project" "../$project"
+done
+
+## Sanity checking....
+declare -x IN_DOCKER=false IN_VSCODE=false
+
+### If we are running docker, then the container knows what to do
+if [[ -f .dockerenv ]]; then
+    export IN_DOCKER=true;
+fi
+
+### Same thing as vscode; the extension handles setting things up
+if ${VSCODE:-false}; then
+    export IN_VSCODE=true;
+fi
+
+### Exit early if setup is handled by others
```

Comment:
I'll need this mechanism explained to me. The difference of running in Docker, Vs Code or neither. Maybe we can get on a call tomorrow?